notebook.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "AiZynthFinder.ipynb",
+      "private_outputs": true,
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ObD2YL7nM2_X"
+      },
+      "source": [
+        "# AiZynthFinder\n",
+        "\n",
+        "Click the ▶ play button at the left of the **Start** text below to run the application. The initial installation process may take a few minutes.\n",
+        "\n",
+        "1. Enter the target compound [SMILES][1] code.\n",
+        "3. Click the **Run Search** button to start the algorithm.\n",
+        "4. Once it stops serching, click the **Show Reactions** button.\n",
+        "\n",
+        "[1]: https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "beDP-CSNM429"
+      },
+      "source": [
+        "#@title Start {display-mode: \"form\"}\n",
+        "!curl --location --silent bit.ly/rdkit-colab | tar xz -C /\n",
+        "!pip install --quiet graphviz\n",
```

Comment:
How does this install the graphviz binaries? Doesn't this only install the python package called graphviz? 
I know the graphviz binaries were installed somehow because the route images were generated, I just would like to know how

Reply:
Actually, Google Colaboratory has `graphviz` installed by default, but it does not seem to be officially documented anywhere. If we ever need to explicitly install it, `!apt install --yes graphviz` would be enough.

Reply:
So then this line could be remove.

Reply:
The [existing] graphviz line (which is for Python) should not be removed, but the `apt` line on my comment (which is of the `dot` binary and so on) shouldn't be added neither, as we already have it installed.